tests/MyGame/Example/Color.d

```diff
@@ -0,0 +1,11 @@
+// automatically generated, do not modify
+
+module MyGame.Example.Color;
+
+enum Color : byte
+{
+   Red = 1,
```

Comment:
it comes from the defintion in the flatbuffers file. It should not force it to D style as some people prefer uppercase enum members